src/Tribe/Views/V2/View.php

```diff
@@ -793,8 +805,9 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$prev_page  = $this->repository->prev()->order_by( '__none' );
 
-		$paged      = $this->url->get_current_page() - 1;
-		$query_args = $paged > 1
+		$paged           = $this->url->get_current_page() - 1;
+		$query_args      = [];
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 393d1683914e0d780296d8643b6adbe6</sub>